src/common/snippets/include/snippets/lowered/linear_ir.hpp

```diff
@@ -7,6 +7,7 @@
 #include <list>
 
 #include "expression.hpp"
+#include "snippets/target_machine.hpp"
```

Comment:
Minor: looks like this including is useless here. Could you please remove or move to source file if needed? Thanks!

Reply:
Actually we need it since `TargetMachine` is a parameter of `init_emitters` [method](https://github.com/openvinotoolkit/openvino/blob/e18377be2514a378001ad905716352b8830ba958/src/common/snippets/include/snippets/lowered/linear_ir.hpp#L98). Or you suggest using forward declaration instead of include in hpp file?

Reply:
Oh I see that now this is parameter of one of methods. I didn't find it before. Nice, thanks!